instrumentation-api-incubator/src/main/jflex/SqlSanitizer.jflex

```diff
@@ -52,6 +54,10 @@ WHITESPACE          = [ \t\r\n]+
   // max length of the sanitized statement - SQLs longer than this will be trimmed
   static final int LIMIT = 32 * 1024;
 
+  // Match on "IN(?, ?, ...)"
+  private static final Pattern IN_STATEMENT_PATTERN = Pattern.compile("(\\sin\\s*)\\(\\s*\\?\\s*(,\\s*\\?\\s*)*+\\)", Pattern.CASE_INSENSITIVE);
+  private static final String IN_STATEMENT_NORMALIZED = " in(?)";
```

Comment:
```suggestion
  private static final String IN_STATEMENT_NORMALIZED = "$1(?)";
```
Sanitizer does not change case or remove whitespace from the original query. Lets keep `in` as it was in the original query. `longInStatementDoesntCauseStackOverflow` will break after this change as there is a space between in and ( that is currently removed.

Reply:
Got it, I updated it to preserve case and whitespace, and updated the test cases to check for that. I didn't add a test case for more than one space between `IN` and `(` since strings like `IN   (?)` get sanitized to `IN (?)`

Also switched to a non-capturing group for matching on the part in-between the brackets as a small optimization